docs/spec_draft.md

```diff
@@ -485,6 +486,71 @@ IEEE-754 specification.
 
 [Back to Ops](#index-of-ops)
 
+## stablehlo.compare
+
+### Semantics
+
+Performs element-wise comparison of `lhs` and `rhs` tensors according to
+`comparison_direction` and `compare_type`, and produces a `result` tensor. More
+formally, `result[i0, ..., iR-1]` =
+`comparison_direction lhs[i0, ..., iR-1], rhs[i0, ..., iR-1]`.
+
+`comparison_direction` can be one of the following:
+  * `EQ`: `lhs` $=$ `rhs`.
+  * `NE`: `lhs` $\ne$ `rhs`.
+  * `GE`: `lhs` $\ge$ `rhs`.
+  * `GT`: `lhs` $\gt$ `rhs`.
+  * `LE`: `lhs` $\le$ `rhs`.
+  * `LT`: `lhs` $\lt$ `rhs`.
+
+`compare_type` can be one of the following:
+  * `FLOAT`: Standard floating-point comparison of `lhs` and `rhs` as specified
```

Comment:
Just for clarity:
`FLOAT`: Ordered floating point comparison of .... 

means that if any one of the operand is a NaN it return would be false. This goes with the spec
```
Four mutually exclusive relations are possible: less than, equal, greater than, and unordered. The last case
arises when at least one operand is NaN.
```
That is if any of the operand is NaN, then the result of compare is `unordered`, which is not equal to other options. 

Also the behavior of the evaluator:
```
func.func @main() -> tensor<i1> {
  %0 = mhlo.constant dense<0x7fc00000> : tensor<f32>
  %1 = mhlo.constant dense<7.5> : tensor<f32>
  %2 = "mhlo.compare"(%0, %1) {compare_type = #mhlo<comparison_type FLOAT>, comparison_direction = #mhlo<comparison_direction XXX>}   :(tensor<f32>, tensor<f32>) -> tensor<i1>
  func.return %2 : tensor<i1>
}

the program will be evaluated to false irrespectove of the value of XXX.
```

Reply:
Thanks